### PR TITLE
Cleanup older proj < 6 scripts and objects, deprecate vw_srs  from srs db

### PR DESCRIPTION
**Pull request by nyalldawson**
Cleanup a bunch of older proj < 6 compatibility code, and move away from the hardcoded list of projection names from srs db to translated names.

### CODE REVIEW

File: src/core/proj/qgscoordinatereferencesystem.cpp [hunk 1]
```diff
@@ -2616,6 +2617,18 @@ int QgsCoordinateReferenceSystem::syncDatabase()
...
 
+      // there's a not-null contraint on these columns, so we must use empty strings instead
+      QString operation = "";
+      QString ellps = "";
+      getOperationAndEllipsoidFromProjString( proj4, operation, ellps );
+
+      const QString translatedOperation = QgsCoordinateReferenceSystemUtils::translateProjection( operation );
+      if ( translatedOperation.isEmpty() && !operation.isEmpty() )
+      {
+        std::cout << QStringLiteral( "Operation needs translation in QgsCoordinateReferenceSystemUtils::translateProjection: %1" ).arg( operation ).toLocal8Bit().constData() << std::endl;
```
Comment (nirvn) [inline src/core/proj/qgscoordinatereferencesystem.cpp:2628]: Deliberately not using QgsDebugMsg here?

File: src/gui/qgsprojectionselectiontreewidget.cpp [hunk 2]
```diff
@@ -655,25 +656,28 @@ void QgsProjectionSelectionTreeWidget::loadCrsList( QSet<QString> *crsFilter )
...
+        QString projectionName;
+        if ( projectionAcronym.isEmpty() )
+          projectionName = tr( "Other" );
+        else
+          projectionName = QgsCoordinateReferenceSystemUtils::translateProjection( projectionAcronym );
 
         // Find the node for this type and add the projection to it
         // If the node doesn't exist, create it
-        if ( srsType == previousSrsType )
+        if ( projectionName == previousSrsType )
```
Comment (nirvn) [inline src/gui/qgsprojectionselectiontreewidget.cpp:668]: I'd change the previousSrsType variable to previousProjectionName, easier to read for people who will visit the code in years to come :)

File: src/core/proj/qgscoordinatereferencesystem.cpp (hunk 1 above)
Comment (nyalldawson) [reply src/core/proj/qgscoordinatereferencesystem.cpp:2628]: Yep -- it won't show up in the cli output from crssync.

Comment (nirvn) [reply src/core/proj/qgscoordinatereferencesystem.cpp:2628]: @nyalldawson , roger that.

File: src/gui/qgsprojectionselectiontreewidget.cpp (hunk 2 above)
Comment (nyalldawson) [reply src/gui/qgsprojectionselectiontreewidget.cpp]: Done (but actually this code is about to be moved to the ether anyway!)

**Review by nirvn — APPROVED** [review]

**Pull request merged**